### PR TITLE
clc_loadbalancer: PEP8 compliancy and documentation changes

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This PR includes:
- PEP8 compliancy changes
- Documentation changes

##### ISSUE TYPE
 - Feature Pull Request
 - Docs Pull Request

##### COMPONENT NAME
clc_loadbalancer

##### ANSIBLE VERSION
v2.5

### CODE REVIEW

File: lib/ansible/modules/cloud/centurylink/clc_loadbalancer.py [hunk 1]
```diff
@@ -866,23 +853,16 @@ def define_argument_spec():
...
-            location=dict(required=True),
-            alias=dict(required=True),
-            port=dict(choices=[80, 443]),
-            method=dict(choices=['leastConnection', 'roundRobin']),
-            persistence=dict(choices=['standard', 'sticky']),
+            name=dict(type='str', required=True),
+            description=dict(type='str'),
+            location=dict(type='str', required=True),
+            alias=dict(type='str', required=True),
+            port=dict(type='str', choices=[80, 443]),
```
Comment (resmo) [inline lib/ansible/modules/cloud/centurylink/clc_loadbalancer.py:860]: the default is `type='str'`, so it was `str` before and that is why I agree this is a no change. But I wonder if this shouldn't be a `type='int'`?

Comment (dagwieers) [reply lib/ansible/modules/cloud/centurylink/clc_loadbalancer.py:860]: I agree. I noticed the same thing when editing, however I did not want to change the behaviour of the module in this PR.

One benefit of always specifying the type for parameters is that such issues become more visible. Especially older modules suffer from this.

**Comment by resmo** [comment]
@dagwieers needs rebase

**Comment by gundalow** [comment]
As agreed in 2017-12-07 Core meeting, we've done bulk pep8 fixes across all modules in `devel`.

Closing this PR as the pep8 fixes have been done by the following PRs:
https://github.com/ansible/ansible/pull/33682
https://github.com/ansible/ansible/pull/33690

**Pull request closed without merging**